backend/src/fcgimain.c

```diff
@@ -805,7 +805,7 @@ static void fcgi_delanswerandfollowing(struct kreq *req)
     }
     else if (question&&question->val) {
       // No answer give, so delete all answers to the given question
-      if (session_delete_answers_by_question_uid(s,question->val,0)<0) {
+      if (session_delete_answers_by_question_uid(s,question->val,1)<0) {
```

Comment:
What is the meaning of this change?

Reply:
This is old... I turns on deleting of successor answers by default, effectively rolling back the survey state to that answer uid. All questions after that uid have to be reanswered (they provide old answer values as default value)
Otherwise nextquestion logic has trouble to deal with changed answer values

since #462 this flag is removed alltogether